.github/workflows/go.yaml

```diff
@@ -1,10 +1,10 @@
-on: [push]
+on: [push, pull_request]
 jobs:
   go-build:
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        go: ["1.14", "1.13", "1.12"]
+        go: ["1.16"]
```

Comment:
While it's true that we probably don't need to be supporting Go 1.12 or 1.13, I'm wary of dropping support for everything but the latest version. I believe the two most recent Go versions are officially supported. This might make a little more sense once Go 1.17 is out.

Reply:
Walking around and reminding that Go 1.17 was released.